SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,809 @@
+
+  <div class="mySlides fade">
+    <img src="https://bootstrapmade.com/content/demo/NiceRestaurant/assets/img/restaurant/misc-square-3.webp" style="width:100%">
+  </div>
+<!-- Next and previous buttons -->
+  <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+  <a class="next" onclick="plusSlides(1)">&#10095;</a>
+</div>
+<br>
+    </section>
+  </main>
+  </header-bar-section>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </header-bar-section> ]

(tag-pair)